src/components/SecondaryPanes/Scopes.css

```diff
@@ -29,6 +29,10 @@
   overflow: auto;
 }
 
+.scopes-list {
+  padding-left: 4px;
```

Comment:
not sure how we deal with LTR languages, but `padding-inline-start` might be a bit better

Reply:
Oh, that's interesting!

Reply:
we should lint for `padding-left` `padding-right`

Reply:
@nchevobbe @jasonLaster we use https://github.com/gasolin/postcss-bidirection as way to handle ltr & rtl which polyfills ...start into ...left or ...right depending on the direction.